TM1py/Services/HierarchyService.py

```diff
@@ -211,6 +211,30 @@ def remove_all_edges(self, dimension_name: str, hierarchy_name: str = None, **kw
         }
         return self._rest.PATCH(url=url, data=json.dumps(body), **kwargs)
 
+    def remove_edges_under_consolidation(self, dimension_name: str,  hierarchy_name: str,
+                                         consolidation_element: str, **kwargs) -> Response:
+        """
+        :param dimension_name: Name of the dimension
+        :param hierarchy_name: Nmae of the hierarchy
+        :param consolidation_element: Name of the Consolidated element
+        :return: response
+        """
+        hierarchy = self.get(dimension_name, hierarchy_name)
+        from TM1py.Services import ElementService
+        element_service = ElementService(self._rest)
+        elements_under_consolidations = element_service.get_members_under_consolidation(dimension_name, hierarchy_name,
+                                                                                        consolidation_element)
+        elements_under_consolidations.append(consolidation_element)
+        remove_edges = []
+        for element, _ in hierarchy.edges.items():
+            if element[0] in elements_under_consolidations and element[1] in elements_under_consolidations:
+                remove_edges.append(element)
+        hierarchy.remove_edges(remove_edges)
+
+        url = format_url("/api/v1/Dimensions('{}')/Hierarchies('{}')", dimension_name, hierarchy_name)
```

Comment:
I think you can just use the `self.update` function here instead

Reply:
agreed..